include/zephyr/net/net_context.h

```diff
@@ -703,6 +709,17 @@ static inline void net_context_set_ipv4_ttl(struct net_context *context,
 	context->ipv4_ttl = ttl;
 }
 
+static inline uint8_t net_context_get_ipv4_mcast_ttl(struct net_context *context)
```

Comment:
there are quite a few of these `net_context_get|set_*` functions (existing ones, and the ones you added) that are missing doxygen comments. Could you please add them? Should these be considered internal (I have no idea myself :)) you may consider not having the doxygen documentation (although it's always nice to have imo), and simply enclose them in ` @cond INTERNAL_HIDDEN` sections.

Sorry that I didn't notice earlier as I really just paid attention to the migration guide changes when the bot added me.